x/evm/keeper/grpc_query_test.go

```diff
@@ -1,26 +1,53 @@
 package keeper_test
 
 import (
+	"crypto/ecdsa"
 	"encoding/json"
-	"fmt"
+	"math/big"
+	"regexp"
 
 	"cosmossdk.io/math"
+	"github.com/NibiruChain/collections"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	gethcommon "github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
-
-	"github.com/NibiruChain/collections"
+	"github.com/ethereum/go-ethereum/crypto"
+	gethparams "github.com/ethereum/go-ethereum/params"
 
 	srvconfig "github.com/NibiruChain/nibiru/app/server/config"
 	"github.com/NibiruChain/nibiru/eth"
-	"github.com/NibiruChain/nibiru/x/common"
 	"github.com/NibiruChain/nibiru/x/common/testutil/testapp"
 	"github.com/NibiruChain/nibiru/x/evm"
 	"github.com/NibiruChain/nibiru/x/evm/evmtest"
 )
 
 func InvalidEthAddr() string { return "0x0000" }
 
+func TraceNibiTransfer() string {
+	return `{
+	  "gas": 21000,
```

Comment:
Why is it this number specifically? Rather, can you replace this with the causal constant from "gethparams" and use `fmt.Sprintf`?